include/irr/asset/filters/dithering/CPrecomputedDither.h

```diff
@@ -58,11 +102,15 @@ namespace irr
 
 					private:
 
+						static constexpr auto decodeFormat = EF_R64G64B64A64_SFLOAT;
```

Comment:
if the definition of `get` is `float` then your decode format is 32-bit float not 64bit ;)

Reply:
Oh okay

Reply:
you want to use `floats` instead of `double`s basically.

Reply:
corrected in following commit